### PR TITLE
LIVE-1092: Offline premium taster

### PR DESCRIPTION
**Pull request by faresite**
This is offline premium taster indicator that builds on top of [PR](https://github.com/guardian/mobile-apps-article-templates/pull/1267) 

| Light | Dark|
| --- | --- |
|||

### CODE REVIEW

File: ArticleTemplates/assets/js/modules/premiumTaster.js [hunk 1]
```diff
@@ -23,51 +23,106 @@ const svg = `
         </g>
     </svg>
 `
+
+let offlineSvg = `
```
Comment (webb04) [inline ArticleTemplates/assets/js/modules/premiumTaster.js:27]: Can this be a const?

File: ArticleTemplates/assets/js/modules/premiumTaster.js [hunk 2]
```diff
@@ -23,51 +23,106 @@ const svg = `
...
+    window.offlineTasterSetup = offlineTasterSetup
+    window.indicatorHtml  = indicatorHtml
 }
 
-function adFreeTasterSetup() {
+function indicatorHtml(taster) {
+    console.log("before check")
+    if (taster !== "adFree" && taster !== "offline") return "";
+    console.log("after check")
+    var text = ""
```
Comment (webb04) [inline ArticleTemplates/assets/js/modules/premiumTaster.js:61]: And these be `let`? It looks like valid swift too with `let` + `var` 😀

File: ArticleTemplates/assets/js/modules/premiumTaster.js [hunk 3]
```diff
@@ -23,51 +23,106 @@ const svg = `
...
+            <rect x="11.3158" y="61.7559" width="69.4737" height="3.94737" transform="rotate(-45 11.3158 61.7559)" fill="#001536"/>
+            <rect x="14.107" y="64.5469" width="69.4737" height="3.94737" transform="rotate(-45 14.107 64.5469)" fill="#C1D8FC"/>
+        </g>
+    </svg>
+`
+
+const signalPaths = {
```
Comment (webb04) [inline ArticleTemplates/assets/js/modules/premiumTaster.js:41]: This is nice 👍

File: ArticleTemplates/assets/js/modules/premiumTaster.js [hunk 4]
```diff
@@ -23,51 +23,106 @@ const svg = `
...
+}
 function init() {
     window.adFreeTasterSetup = adFreeTasterSetup
+    window.offlineTasterSetup = offlineTasterSetup
+    window.indicatorHtml  = indicatorHtml
 }
 
-function adFreeTasterSetup() {
+function indicatorHtml(taster) {
+    console.log("before check")
```
Comment (webb04) [inline ArticleTemplates/assets/js/modules/premiumTaster.js:58]: Couple of checks left in

**Review by webb04 — APPROVED** [review]
Good job, lgtm! 🎉

File: ArticleTemplates/assets/js/modules/premiumTaster.js (hunk 2 above)
Comment (faresite) [reply ArticleTemplates/assets/js/modules/premiumTaster.js:61]: Gosh I've been really swiftified haven't I!

**Review by webb04 — APPROVED** [review]

**Pull request merged**